openfish-webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Openfish Webapp
+Openfish Webapp is a web interface for interacting wit the Openfish API.
+
+## Development guide
+#### Getting started
+
+Run the commands:
+```
+cd openfish-webapp
```

Comment:
We're assuming their working directory is api ?

Reply:
I'm assuming their working directory is the root of the git repo